neurodsp/plts/time_series.py

```diff
@@ -19,9 +19,9 @@ def plot_time_series(times, sigs, labels=None, colors=None, ax=None, **kwargs):
 
     Parameters
     ----------
-    times : 1d array or list of 1d array
+    times : 1d array, list of 1d array, or 2d array
         Time definition(s) for the time series to be plotted.
-    sigs : 1d array or list of 1d array
+    sigs : 1d array, list of 1d array, or 2d array
```

Comment:
probably this can be condense to say something like:
`1d or 2d array or list of 1d array`
I'm not sure if there's an exact convention for specifying a range of array sizes - but one way or another the array_type input can probably be grouped, and same for the `times` input